Source/JavaScriptCore/b3/air/AirOpcode.opcodes

```diff
@@ -1872,9 +1872,15 @@ arm64: VectorBitmask U:G:8, U:F:128, ZD:G:32
 x86_64: VectorBitmask U:G:8, U:F:128, ZD:G:32, S:F:128
     SIMDInfo, Tmp, Tmp, Tmp
 
-VectorExtaddPairwise U:G:8, U:F:128, D:F:128
+arm64: VectorExtaddPairwise U:G:8, U:F:128, D:F:128
     SIMDInfo, Tmp, Tmp
 
+x86_64: VectorExtaddPairwise U:G:8, U:F:128, D:F:128, S:G:64, S:F:128
+    SIMDInfo, Tmp, Tmp, Tmp, Tmp
+
+x86_64: VectorExtaddPairwise U:F:128, D:F:128, S:F:128, S:F:128
```

Comment:
If we do not have SIMDInfo, let's add more specific lane, signedness etc. information to this instruction name.